File: Curso-de-POO/Python/payment.py
class Payment:
    id = int
    
    def __init__(self, id):
        self.id = id


class Card(Payment):
    number = str
    CVV = str
    date = str

    def __init__(self, number, CVV, date):
        super().__init__(id)
        self.number = number
        self.CVV = CVV
        self.date = date


class PayPal(Payment):
    email = str

    def __init__(self, id, email):
        super().__init__(id)
        self.email = email


class Cash(Payment):

    def __init__(self, id):
        super().__init__(id)

File: Curso-de-POO/Java/Payment.java
class Payment {
    Integer id;

    public Payment(Integer id){
        this.id = id;
    }
}

class Card  extends Payment {
    String number;
    String CVV;
    String date;

    public Card(Integer id, String number, String CVV, String date) {
        super(id);
        this.number = number;
        this.CVV = CVV;
        this.date = date;
    }
}

class Paypal extends Payment {
    String email;
    
    public Paypal(Integer id, String email){
        super(id);
        this.email = email;
    }
}

class Cash extends Payment {
    public Cash(Integer id){
        super(id);
    }
}
